From 2a9b4169a25273dcf3c474c76455ab31d1e2839b Mon Sep 17 00:00:00 2001
From: Leah Rowe <info@minifree.org>
Date: Sun, 14 Jun 2015 15:40:00 +0100
Subject: [PATCH 8/9] lenovo/t500: Add clone of Lenovo T400

The existing code for the Lenovo T400 works without changes on the
Lenovo T500.  Same HDA verbs are provided by Lenovo BIOS on both
laptops.

Change-Id: I300408a8a0ed00476aee6061925befc2822fb505
Signed-off-by: Leah Rowe <info@minifree.org>
---
 src/mainboard/lenovo/t400/Kconfig        | 2 +-
 src/mainboard/lenovo/t500/Kconfig        | 7 +++++++
 src/mainboard/lenovo/t500/Kconfig.name   | 2 ++
 src/mainboard/lenovo/t500/board_info.txt | 6 ++++++
 4 files changed, 16 insertions(+), 1 deletion(-)
 create mode 100644 src/mainboard/lenovo/t500/Kconfig
 create mode 100644 src/mainboard/lenovo/t500/Kconfig.name
 create mode 100644 src/mainboard/lenovo/t500/board_info.txt

diff --git a/src/mainboard/lenovo/t400/Kconfig b/src/mainboard/lenovo/t400/Kconfig
index 467cd63..0eac311 100644
--- a/src/mainboard/lenovo/t400/Kconfig
+++ b/src/mainboard/lenovo/t400/Kconfig
@@ -1,4 +1,4 @@
-if BOARD_LENOVO_T400 || BOARD_LENOVO_R400
+if BOARD_LENOVO_T400 || BOARD_LENOVO_T500 || BOARD_LENOVO_R400
 
 config BOARD_SPECIFIC_OPTIONS # dummy
 	def_bool y
diff --git a/src/mainboard/lenovo/t500/Kconfig b/src/mainboard/lenovo/t500/Kconfig
new file mode 100644
index 0000000..e1e8420
--- /dev/null
+++ b/src/mainboard/lenovo/t500/Kconfig
@@ -0,0 +1,7 @@
+if BOARD_LENOVO_T500
+
+config MAINBOARD_PART_NUMBER
+	string
+	default "ThinkPad T500"
+
+endif
diff --git a/src/mainboard/lenovo/t500/Kconfig.name b/src/mainboard/lenovo/t500/Kconfig.name
new file mode 100644
index 0000000..89c6087
--- /dev/null
+++ b/src/mainboard/lenovo/t500/Kconfig.name
@@ -0,0 +1,2 @@
+config BOARD_LENOVO_T500
+	bool "ThinkPad T500"
diff --git a/src/mainboard/lenovo/t500/board_info.txt b/src/mainboard/lenovo/t500/board_info.txt
new file mode 100644
index 0000000..007ec6c
--- /dev/null
+++ b/src/mainboard/lenovo/t500/board_info.txt
@@ -0,0 +1,6 @@
+Category: laptop
+ROM package: SOIC-16 or SOIC-8
+ROM protocol: SPI
+ROM socketed: n
+Flashrom support: n
+Clone of: lenovo/t400
-- 
1.9.1

